packages/slate-editor/src/modules/editor/types.ts

```diff
@@ -105,6 +106,7 @@ export interface EditorProps {
     withGalleries?: false | GalleriesExtensionConfiguration;
     withHeadings?: boolean;
     withImages?: false | ImageExtensionConfiguration;
+    withInlineContacts?: false | InlineContactsExtensionParameters;
```

Comment:
Is this a good name or do you think it should be different?

Reply:
> I am still not sure about the name as inline to me implies something that can be inlined in a text (like inline links).
> 
> Thoughts?

I share your concern. But I can't come with any name that would be better and still remain clear for us, developers. 

Other options I have:
- ad-hoc contacts?
- local contacts?
- contact cards?

But I'm not liking any of the above, more than "inline contacts".

:shrug:

Reply:
I like `contact-cards` but it's not very clear for us, since we have two variations 🤔

Reply:
> but it's not very clear for us, since we have two variations

:100: Exactly my thoughts.